packages/util/src/array.spec.ts

```diff
@@ -120,3 +122,43 @@ describe('array', () => {
     });
   });
 });
+
+describe('zip', () => {
```

Comment:
perhaps add a test where one array is really  a few elements longer than the other one to see that it is just appended. From the word `zip` with the docs it is clear, but the examples look as if the method is sorting. Perhaps you call the elements so that the result is not sorterd.

Reply:
Will do

Reply:
Done.